CIP-0013/CIP-0013.md

```diff
@@ -43,27 +55,66 @@ Example:
 
+### Stake pool URI queries
+
+For brevity, essential in many Internet contexts, `poolticker`  must be supported here in addition to the unambiguous `poolhexid`.
+
+When there is more than one pool registered with the specified `poolTicker` (whether for pool groups which have the same ticker for all pools, or for separate pools using the same ticker), the choice to which pool to finally delegate is left to the user through the wallet UI.
+
+### Handling stake pool links
+
+The wallet UI should always confirm the exact delegation choice even when it is unambiguous from the URI.  When the user has multiple wallets, the wallet UI must select which wallet(s) the user will be delegating from.
+
+These yet unsupported parameters in the URI query string should (by preference of the wallet UI designers) *either* be ignored *or* generate a warning message, to avoid leading the user to believe they are implementing a currently unsupported but perhaps popularly referenced multi-pool delegation list:
+
+* any value for the first URI query argument;
+* any URI query argument beyond the first.
+
 ## Security Considerations
 
-1. We cannot prompt the user to send the funds right away as they may not be fully aware of the URI they clicked or were redirected to. Instead, it may be better to simply pre-populate fields in a transaction.
-2. We should be wary of people who disguise “donate” links as actually opening up a phishing website that LOOKS like a wallet.
+1. For payment links, we cannot prompt the user to send the funds right away as they may not be fully aware of the URI they clicked or were redirected to. Instead, it may be better to simply pre-populate fields in a transaction.
+2. For either payment or staking links, we should be wary of people who disguise links as actually opening up a phishing website that LOOKS like that corresponding part of the wallet UI.
 
 # Rationale
 
 ## Why not use Universal links, deep links and other non-protocol-based Solution
 
 An alternative solution to the original problem described above is to use standard URL links in combination with a routing backend system. The routing system is used to redirect to the app's URI. The advantage of this scheme is that it allows to provide a fallback mechanism to handle the case when no application implementing the protocol is installed (for instance, by redirecting to the App Store or Google Play). This is the approach behind iOS Universal Links and Android App Links. In general, it provides a better user experience but requires a centralized system which makes it unsuitable for as a multi-app standard.
 
+## Why stake pool links are important
+
+As K continues to increase, options for convenient and informed re-delegation will be more important as Ada holders face pool saturation more often. Interfaces that connect delegators with pools beyond the highly contested top choices of the in-wallet ranking algorithms are important to avoid saturation and maintain decentralization.
```

Comment:
You'll notice I avoided mentioning K in my wording specifically because K will not be increasing forever , and this specification should remain relevant for years. You could mention instead something like "as the number of entities interested in running a stake pool continues to increase" as this is hopefully relevant for a long time

Similarly, "Ada holders face pool saturation more often" is just a temporary symptom of K increasing and should reach equilibrium so I don't think it's really relevant to this CIP.

Reply:
Removed first sentence about K, to re-post in this PR as per my general comment.